### PR TITLE
fix: min max age filter logic

### PR DESCRIPTION
**Pull request by melniiv**
## Description
Replace audience age params with opposite to fix the search query

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by nikomakela** [comment]
The PR-test-server throws an error from the proxy and cannot fetch the events:

```
{"errors":[{"message":"Unknown argument \"audienceMaxAgeGt\" on field \"Query.eventList\". Did you mean \"audienceMaxAgeLt\" or \"audienceMinAgeGt\"?","locations":[{"line":3,"column":5}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","exception":{"stacktrace":["GraphQLError: Unknown argument \"audienceMaxAgeGt\" on field \"Query.eventList\". Did you mean \"audienceMaxAgeLt\" or \"audienceMinAgeGt\"?","    at Object.Argument (/app/node_modules/graphql/validation/rules/KnownArgumentNamesRule.js:46:29)","    at Object.enter (/app/node_modules/graphql/language/visitor.js:323:29)","    at Object.enter (/app/node_modules/graphql/utilities/TypeInfo.js:370:25)","    at visit (/app/node_modules/graphql/language/visitor.js:243:26)","    at Object.validate (/app/node_modules/graphql/validation/validate.js:69:24)","    at validate (/app/node_modules/apollo-server-core/dist/requestPipeline.js:233:34)","    at Object.<anonymous> (/app/node_modules/apollo-server-core/dist/requestPipeline.js:119:42)","    at Generator.next (<anonymous>)","    at fulfilled (/app/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)","    at processTicksAndRejections (internal/process/task_queues.js:97:5)"]}}},{"message":"Unknown argument \"audienceMinAgeLt\" on field \"Query.eventList\". Did you mean \"audienceMinAgeGt\" or \"audienceMaxAgeLt\"?","locations":[{"line":4,"column":5}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","exception":{"stacktrace":["GraphQLError: Unknown argument \"audienceMinAgeLt\" on field \"Query.eventList\". Did you mean \"audienceMinAgeGt\" or \"audienceMaxAgeLt\"?","    at Object.Argument (/app/node_modules/graphql/validation/rules/KnownArgumentNamesRule.js:46:29)","    at Object.enter (/app/node_modules/graphql/language/visitor.js:323:29)","    at Object.enter (/app/node_modules/graphql/utilities/TypeInfo.js:370:25)","    at visit (/app/node_modules/graphql/language/visitor.js:243:26)","    at Object.validate (/app/node_modules/graphql/validation/validate.js:69:24)","    at validate (/app/node_modules/apollo-server-core/dist/requestPipeline.js:233:34)","    at Object.<anonymous> (/app/node_modules/apollo-server-core/dist/requestPipeline.js:119:42)","    at Generator.next (<anonymous>)","    at fulfilled (/app/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)","    at processTicksAndRejections (internal/process/task_queues.js:97:5)"]}}}]}
```

**Comment by nikomakela** [comment]
> The PR-test-server throws an error from the proxy and cannot fetch the events:
> 
> ```
> {"errors":[{"message":"Unknown argument \"audienceMaxAgeGt\" on field \"Query.eventList\". Did you mean \"audienceMaxAgeLt\" or \"audienceMinAgeGt\"?","locations":[{"line":3,"column":5}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","exception":{"stacktrace":["GraphQLError: Unknown argument \"audienceMaxAgeGt\" on field \"Query.eventList\". Did you mean \"audienceMaxAgeLt\" or \"audienceMinAgeGt\"?","    at Object.Argument (/app/node_modules/graphql/validation/rules/KnownArgumentNamesRule.js:46:29)","    at Object.enter (/app/node_modules/graphql/language/visitor.js:323:29)","    at Object.enter (/app/node_modules/graphql/utilities/TypeInfo.js:370:25)","    at visit (/app/node_modules/graphql/language/visitor.js:243:26)","    at Object.validate (/app/node_modules/graphql/validation/validate.js:69:24)","    at validate (/app/node_modules/apollo-server-core/dist/requestPipeline.js:233:34)","    at Object.<anonymous> (/app/node_modules/apollo-server-core/dist/requestPipeline.js:119:42)","    at Generator.next (<anonymous>)","    at fulfilled (/app/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)","    at processTicksAndRejections (internal/process/task_queues.js:97:5)"]}}},{"message":"Unknown argument \"audienceMinAgeLt\" on field \"Query.eventList\". Did you mean \"audienceMinAgeGt\" or \"audienceMaxAgeLt\"?","locations":[{"line":4,"column":5}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","exception":{"stacktrace":["GraphQLError: Unknown argument \"audienceMinAgeLt\" on field \"Query.eventList\". Did you mean \"audienceMinAgeGt\" or \"audienceMaxAgeLt\"?","    at Object.Argument (/app/node_modules/graphql/validation/rules/KnownArgumentNamesRule.js:46:29)","    at Object.enter (/app/node_modules/graphql/language/visitor.js:323:29)","    at Object.enter (/app/node_modules/graphql/utilities/TypeInfo.js:370:25)","    at visit (/app/node_modules/graphql/language/visitor.js:243:26)","    at Object.validate (/app/node_modules/graphql/validation/validate.js:69:24)","    at validate (/app/node_modules/apollo-server-core/dist/requestPipeline.js:233:34)","    at Object.<anonymous> (/app/node_modules/apollo-server-core/dist/requestPipeline.js:119:42)","    at Generator.next (<anonymous>)","    at fulfilled (/app/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)","    at processTicksAndRejections (internal/process/task_queues.js:97:5)"]}}}]}
> ```

Relates to https://github.com/City-of-Helsinki/events-helsinki-api-proxy/pull/87

**Review by nikomakela — APPROVED** [review]
The changes looks good, but the result set should be tested carefully when the proxy changes are active.

File: src/domain/search/eventSearch/AdvancedSearch.tsx [hunk 1]
```diff
@@ -177,8 +177,8 @@ const AdvancedSearch: React.FC<Props> = ({
     setSelectedTexts(text || []);
     setEnd(endTime);
     setStart(startTime);
-    setMinAgeInput(audienceMinAgeGt || '');
```
Comment (nenonja) [inline src/domain/search/eventSearch/AdvancedSearch.tsx:180]: min and greater than was previously mixed. Was it a bug?

**Review by nenonja — APPROVED** [review]

File: src/domain/search/eventSearch/AdvancedSearch.tsx (hunk 1 above)
Comment (melniiv) [reply src/domain/search/eventSearch/AdvancedSearch.tsx:180]: yes, the wrong params were used for min and max age (basically we had to switch to opposite) to filter correctly

**Pull request merged**